### PR TITLE
general.downloadVia added for download services

### PR DESCRIPTION
**Pull request by Venipa**

### CODE REVIEW

**Review by ImRodry — CHANGES_REQUESTED** [review]
Please change back the formatting to 4-space tabs

**Comment by Venipa** [comment]
> Please change back the formatting to 4-space tabs

may i add a .vscode folder with tabWidth prettier setting set to 4?
would probably be better for future changes.

**Review by ImRodry — CHANGES_REQUESTED** [review]
Please change to tabs and not 4 spaces so that we can actually see the diff

**Comment by ImRodry** [comment]
In what context would this be used?

**Comment by Venipa** [comment]
> In what context would this be used?

im currently writing a coub presence and found out that you are able to use custom scheme links inside discord.
for example:
strings.downloadVia + " Bridge"

"Download via Bridge" => https://github.com/Venipa/coubdl-bridge (registers custom scheme to directly download videos from coub)

File: src/Presence/general.json [hunk 1]
```diff
@@ -11,6 +11,10 @@
 		"message": "Chapter",
 		"description": ""
 	},
+	"general.downloadVia": {
+		"message": "Download via",
+		"description": ""
+	},
```
Comment (Bas950) [inline src/Presence/general.json:17]: Download using {0}

Might be better, and add a description

**Review by Bas950 — CHANGES_REQUESTED** [review]

**Comment by ImRodry** [comment]
So I'm still a bit confused, would this be a button to download the program that is showing up as your presence? If so why would you need a button to download when you're simply visiting the website?

**Comment by Venipa** [comment]
> So I'm still a bit confused, would this be a button to download the program that is showing up as your presence? If so why would you need a button to download when you're simply visiting the website?

its only a options for users who are using coubdl-bridge, it wont open a webpage.
it opens a internally without a website.

image

**Comment by ImRodry** [comment]
Well alright but again, where would this show up in the presence itself? Is it a button?

**Comment by Venipa** [comment]
> Well alright but again, where would this show up in the presence itself? Is it a button?

ah, yes its a button

coub.com rewrite:
image

**Comment by ImRodry** [comment]
The minimum required users for a Presence to be illegible to have strings is 1000 and Coub has 167, not to mention I don't see any other presence using this so it wouldn't make sense to put it on general. Lastly, I doubt you will be allowed to add that button but that's not up to me to decide. Sorry, but I'll close this for now.

**Pull request closed without merging**